samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java

```diff
@@ -392,7 +393,10 @@ public void sendTypingNotification(View view) {
 
         if (chatThreadAsyncClient != null) {
             try {
-                chatThreadAsyncClient.sendTypingNotification().get();
+                TypingNotificationOptions options = new TypingNotificationOptions();
+                options.setSenderDisplayName("Sender Display Name");
+
+                chatThreadAsyncClient.sendTypingNotification(options).get();
```

Comment:
The get call reads funny here since no variable is receiving the get result. Please comment if there is a good reason.

Reply:
The existing test calling the other method version does this already, and since I don't know much about CompletableFuture in Android I did the same. Do you this can be safely removed without altering the test logic?

Reply:
All the tests in this class always do .get(), even with no variables to save the result. So I'd be inclined to keep the same pattern